src/sections/header.liquid

```diff
@@ -78,26 +80,29 @@
   </header>
 
   <nav role="navigation">
-    <ul class="site-nav">
-      {% for link in linklists[section.settings.main_linklist].links %}
-        {%- assign child_list_handle = link.title | handleize -%}
-        {% if menus[child_list_handle].links != blank %}
-          <li class="site-nav--has-submenu{% if link.active %} site-nav--active{% endif %}">
-            <a href="{{ link.url }}" class="site-nav__link">
+    <ul>
+      {% for link in linklists[section.settings.menu].links %}
+        {% if link.links != blank %}
+          <li>
+            <a href="{{ link.url }}">
               {{ link.title }}
               {% include 'icon-arrow-down' %}
             </a>
-            <ul class="site-nav__submenu">
-              {% for childlink in menus[child_list_handle].links %}
-                <li {% if childlink.active %}class="site-nav--active"{% endif %}>
-                  <a href="{{ childlink.url }}" class="site-nav__link">{{ childlink.title | escape }}</a>
+            <ul>
```

Comment:
Is it okay to leave `<ul>` alone if there are no child links?

Reply:
I see. My question was :

There's a `for` loop on line 92, but we see an `<ul>` on line 91. If there aren't any child links, it won't output the `<li>` but do we still see the parent `<ul>` (of line 91) in the source code?

Reply:
So we actually do the check above: `{% if link.links != blank %}` (meaning, if there are child links), if there aren't childlinks, then it jumps to the `{% else %}` and outputs a single `<li>`. So no you'd never see the parent `<ul>` in this case.

Reply:
It's all clear to me now! I didn't see the condition above. Thanks!